components/brave_wallet_ui/common/reducers/wallet_reducer.ts

```diff
@@ -165,10 +165,11 @@ reducer.on(WalletActions.nativeAssetBalancesUpdated, (state: any, payload: GetNa
   accounts.forEach((account, index) => {
     if (payload.balances[index].success) {
       accounts[index].balance = payload.balances[index].balance
-      accounts[index].fiatBalance = formatFiatBalance(payload.balances[index].balance, state.selectedNetwork.decimals, payload.usdPrice).toString()
+      accounts[index].fiatBalance = payload.usdPrice !== ''
+        ? formatFiatBalance(payload.balances[index].balance, state.selectedNetwork.decimals, payload.usdPrice).toString()
+        : ''
```

Comment:
Does it make sense to use the cached `usdPrice` from the state? Also, what do you think about `payload.usdPrice !== undefined` here? Unless the `usdPrice` is supposed to be rendered as an empty string somewhere, I think it's better to keep the `''` concept specific to unknown balances.

Reply:
Discussed internally - it's a non-issue.